android/gradle.properties

```diff
@@ -2,3 +2,4 @@ org.gradle.jvmargs=-Xmx1536M
 android.enableR8=true
 android.useAndroidX=true
 android.enableJetifier=true
+android.enableDexingArtifactTransform=false
```

Comment:
do czego jest to używane/jaki problem rozwiązuje?

Reply:
https://github.com/flutter/flutter/issues/72185